playbooks/site.yml

```diff
@@ -1,5 +1,4 @@
----
```

Comment:
I don't think this should be removed, and would think that the copyright gumpf should be here too.